### PR TITLE
UIIN-2288: Align instance subjects and series fields with a new API breaking change

### PR DESCRIPTION
**Pull request by mkuklis**
https://issues.folio.org/browse/UIIN-2288

Align instance subjects and series fields with a new API breaking change.

subjects and series fields are now an array of objects:

````json
"properties": {
  "value": {
    "type": "string",
    "description": "Subject heading value"
  },
  "authorityId": { 
    "type": "string",
    "description": "UUID of authority record that controls a subject heading",
    "$ref": "uuid.json"
  }
}
````

### CODE REVIEW

File: package.json
Comment (mkuklis) [inline package.json:47]: I ordered them alphabetically.

File: package.json
Comment (usavkov-epam) [inline package.json:79]: Looks like you forgot to update the interface to support version `1.0`

Comment (mkuklis) [reply package.json:79]: @usavkov-epam good catch! Thank you.

**Review by usavkov-epam — APPROVED** [review]

**Comment by mkuklis** [comment]
@usavkov-epam do you know if this can be merged now or do we still need to wait?

**Comment by usavkov-epam** [comment]
> @usavkov-epam do you know if this can be merged now or do we still need to wait?

AFAIK, we have to wait until the BE changes are merged and then we will get a ready signal from that command in slack I suppose

**Pull request merged**
